code/src/index.css

```diff
@@ -1,13 +1,33 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Roboto', sans-serif;
+  font-family: 'Roboto Mono', monospace;
+  background: #f6e5cf;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+p {
+  margin: 0;
+}
+
+.drawingBuffer {
+  width: 0;
+  height: 0;
+}
+
+video {
+  width: 320px;
+  height: 240px;
+  transform: scaleX(-1); 
+}
+
+@media (min-width: 667px){
+    video {
+    width: 640px;
+    height: 480px;
+  }
+}
```

Comment:
Try not to mix styled components with vanilla css. The styling you have for video here for example, you could use to style the div in BarcodeScanner.js, from:
`<div ref={cameraDivRef} className={className} />`
to:
```
const Camera = styled.div`
  transform: scaleX(-1);
`
<Camera ref={cameraDivRef} className={className} />
```
or similar 💅